Update german translation

NiklasSTG opened the pull request:
I updated the german translations with an automatic python script

NiklasSTG committed:
Update german translation

NiklasSTG committed:
Update translation

bthidev commented:
Hello, thanks for your proposal, can you remove the .vs folder and update the Readme ?

NiklasSTG committed:
Delete .vs directory

NiklasSTG committed:
Update README.md

Updated german translation

NiklasSTG commented:
Hello, thank you for the answer.
.vs should be deleted now and readme should also be included in the PR now.
I took the liberty and changed the credits in the readme from "NiklasYGO" to "NiklasSTG".
NiklasYGO was my discord name in the past and i changed it a while ago, i hope thats okay.

NiklasSTG committed:
Updated german translation

bthidev commented:
seem good to me
